sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/Configs.java

```diff
@@ -176,6 +182,18 @@ public int getReactorNettyMaxConnectionPoolSize() {
         return REACTOR_NETTY_MAX_CONNECTION_POOL_SIZE;
     }
 
+    public int getDirectHttpsResponseTimeoutInSeconds() {
+        return getJVMConfigAsInt(DIRECT_HTTPS_RESPONSE_TIMEOUT_IN_SECONDS, DEFAULT_DIRECT_HTTPS_RESPONSE_TIMEOUT_IN_SECONDS);
```

Comment:
Question: Why the DIRECT prefix ? This is not just applicable when using Direct Transport with Https Transport (vs. TCP) but for all https calls (independent of Gateway or Direct with Https, correct?) If so I would remove the Direct - could otherwise be too confusing

Reply:
This is done, resolving the comment now.